src/parser.rs

```diff
@@ -131,7 +131,7 @@ fn take_conversion_specifier(s: &str) -> Result<(ConversionSpecifier, &str)> {
         space_sign: false,
         force_sign: false,
         width: NumericParam::Literal(0),
-        precision: NumericParam::Literal(6),
+        precision: NumericParam::FromArgument, // Placeholder - must not be returned!
```

Comment:
Can't say I like this overloading of `FromArgument` but considering adding another variant to `NumericParam` would be a breaking change to the API I think I agree this is the least worst option